content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Go"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+{{ blockEnd }}
+
+{{ startTab "Javascript/Typescript"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Python"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Rust"}}
+
+**SDK: Outbound HTTP**
+
+Local: Yes
+
+Cloud: Yes
```

Comment:
+1 to the table suggestion for this.  What about something like:

| Feature | SDK Supported? | Learn More |
|-----|-----|-------|
| Triggers |
| HTTP trigger | Supported | [Reference](/spin/foo)<br>[Tutorial](/spin/foo)<br/>[Routing blog](https://example.com/r)<br />[Basic Example](https://example.com/blah) |
| Redis trigger | Supported | [Reference](/spin/foo)<br>[Tutorial](/spin/foo)<br/>[Dead Letter Queue Example](https://example.com/blah) |
| APIs |
| Outbound HTTP | Supported | [Reference](/spin/foo)<br>[Tutorial](/spin/foo)<br/>[Dog Facts Demo](https://example.com/blah) |
| MySQL | Unsupported | - |

(of course the Learn More could be split into multiple columns if we want to be precise about docs vs blogs vs examples)

(also we might need to crack multi-column cells to do the triggers vs APIs heading but that's not critical)

Reply:
I am also wondering if we accompany this with an "API Guide" which has the same information but tabbed by API rather than by language, and each tab body talks about cloud and language support and examples, and cross links between that page and this.  But maybe the API pages already cover that?

Which also makes me wonder: should this info be on each language page, either instead of or as well as in the tabbed block?  We don't a reader to have to jump back here when they are already on the language page...?